jetson/filter/src/__main__.py

```diff
@@ -120,7 +124,8 @@ def __init__(self):
             self.config = json.load(config)
 
         self.gps = Gps()
-        self.imu = Imu(self.config["IMU_accel_filter_bias"], self.config["IMU_accel_threshold"])
+        self.imu = Imu(self.config["IMU_accel_filter_bias"],
+                       self.config["IMU_accel_threshold"])
         self.nav_state = None
         self.static_nav_states = {"Off", "Done", "Search Spin Wait", "Turned to Target Wait", "Gate Spin Wait",
```

Comment:
these are no longer correct right?

Reply:
Here are the correct status names:

```c++
// Gets the string representation of a nav state.
std::string StateMachine::stringifyNavState() const {
    static const std::unordered_map<NavState, std::string> navStateNames =
            {
                    {NavState::Off,                  "Off"},
                    {NavState::Done,                 "Done"},
                    {NavState::DriveWaypoints,       "Drive Waypoints"},
                    {NavState::BeginSearch,          "Change Search Algorithm"},
                    {NavState::Search,               "Search"},
                    {NavState::DriveToTarget,        "Drive to Target"},
                    {NavState::TurnAroundObs,        "Turn Around Obstacle"},
                    {NavState::DriveAroundObs,       "Drive Around Obstacle"},
                    {NavState::SearchTurnAroundObs,  "Search Turn Around Obstacle"},
                    {NavState::SearchDriveAroundObs, "Search Drive Around Obstacle"},
                    {NavState::BeginGateSearch,      "Gate Prepare"},
                    {NavState::GateTraverse,         "Gate Traverse"},

                    {NavState::Unknown,              "Unknown"}
            };

    return navStateNames.at(mRover->currentState());
} // stringifyNavState()
```

Reply:
I'm now noticing `BeginSearch` is named wrong

Reply:
that list of static nav states is left over from the KF, it isn't actually used at all rn. on that note, should I just remove all the KF code since we aren't gonna be using it?

Reply:
honestly, yeah, if you chose to rewrite it you can always use the git history as a place to start, or start w smth new